draft-ietf-quic-transport.md

```diff
@@ -2246,6 +2246,16 @@ that packet numbers cannot be used to correlate activity.  This does not prevent
 other properties of packets, such as timing and size, from being used to
 correlate activity.
 
+An endpoint SHOULD NOT initiate migration with a peer that uses a zero-length
+connection ID, because traffic over the new path might be trivially linkable to
+traffic over the old one. If the server is able to route packets with a
+zero-length connection ID to the right connection, it means that the server is
+using other information to demultiplex packets. For example, the server can
+assign an address that is unique to the connection using the preferred_address
+transport parameter; see {{preferred-address}}.  Information that might allow
```

Comment:
Issue #3565 is prohibiting the use of preferred_address with 0-length CID.  So this example is not possible and just confusing (since it is suggesting that the use of preferred_address with 0-length CID is possible).

Reply:
Good catch.  These were done in parallel without thinking much about their interactions.  I've switched to using a different example.